From 54a8b5462e71abbbf4a4c3c444f70abfc6659877 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Sat, 31 Mar 2012 13:17:13 +0200
Subject: [PATCH] usb-redir: Not finding an async urb id is not an error

We clear our pending async urb list on device disconnect and we may still
receive "packet complete" packets from our peer after this, which will then
refer to packet ids no longer in our list.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 usb-redir.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usb-redir.c b/usb-redir.c
index a424826..ec9170a 100644
--- a/usb-redir.c
+++ b/usb-redir.c
@@ -286,7 +286,7 @@ static AsyncURB *async_find(USBRedirDevice *dev, uint32_t packet_id)
             return aurb;
         }
     }
-    ERROR("could not find async urb for packet_id %u\n", packet_id);
+    DPRINTF("could not find async urb for packet_id %u\n", packet_id);
     return NULL;
 }
 
